var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -141,6 +142,10 @@ class Paraview(CMakePackage, CudaPackage):
     # Can't contretize with python2 and py-pillow@7.0.0:
     depends_on('pil@:6', when='+python')
 
+    # ParaView depends on cli11 due to changes in MR
+    # https://gitlab.kitware.com/paraview/paraview/-/merge_requests/4951
+    depends_on('cli11@1.9.1', when='@5.10:')
```

Comment:
should this be >= 5.10?

Reply:
@utkarshayachit That is what `5.10:` does, unless I'm mistaken.

Reply:
I may have misunderstood your question. `5.10:` also includes `paraview@master`, so when the next release happens this won't have to be changed to apply to that version

Reply:
ah! yes, saw the end `:`... thanks for clarifying.